src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_window_request.cc

```diff
@@ -166,6 +166,26 @@ void CreateWindowRequest::Run() {
         (*message_)[strings::msg_params]
                    [strings::duplicate_updates_from_window_id]
                        .asInt();
+
+    const auto default_window_id =
+        mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
+
+    if (default_window_id == duplicate_updates_from_window_id) {
+      const std::string current_stringified_layout =
```

Comment:
@AByzhynar 
Maybe auto ?

Reply:
@SNiukalov Thank you. Done in c5170197a2d095b02c96492eb6a782f406bb9c36